File: tests/compiler/dart/match_expr.dart
int x = 2;

String label = (() {
  var _t = x;
  switch (_t) {
  case 1: return "one";
  case 2: return "two";
  case 3: return "three";
  default: return "unknown";
  }
  return null;
})();

void main() {
  print(label);
}

File: tests/human/x/pl/match_expr.pl
:- initialization(main).
main :-
    X = 2,
    ( X =:= 1 -> Label='one'
    ; X =:= 2 -> Label='two'
    ; X =:= 3 -> Label='three'
    ; Label='unknown'
    ),
    writeln(Label),
    halt.
